src/baseevents.cpp

```diff
@@ -74,6 +74,7 @@ bool BaseEvents::loadFromXml()
 
 		if (!success || !registerEvent(event, node)) {
 			delete event;
+			return false;
```

Comment:
I don't see the reason of this return as it will stop loading in half because an error, thats bad imo, or you load it all and report errors or you load nothing and terminate, the first is as it always was so I wouldn't change that.